src/screens/playing.rs

```diff
@@ -3,7 +3,9 @@
 use bevy::{input::common_conditions::input_just_pressed, prelude::*};
 
 use super::Screen;
-use crate::{asset_tracking::LoadResource, audio::Music, demo::level::SpawnLevel};
+use crate::{
+    asset_tracking::LoadResource, audio::Music, demo::level::spawn_level as spawn_level_command,
```

Comment:
I don't think the type alias is helpful here. A comment would be better.

Reply:
It's needed because we also have function called `spawn_level` here. Imo both of these "deserve" the name, and qualifying this one with `level::spawn_level` does not show what the difference between them is.

Reply:
A point in favor of `level::level` :zipper_mouth_face: